modules/aleph/src/Aleph/Entity/AlephHoldGroup.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Drupal\aleph\Aleph\Entity;
+
+/**
+ * Class AlephHoldGroup.
+ *
+ * Entity for hold groups.
+ *
+ * @package Drupal\aleph\Aleph
+ */
+
+class AlephHoldGroup {
+  protected $subLibrary;
+  protected $subLibraryCode;
+  protected $pickupLocations = [];
+
+  /**
+   * @return mixed
+   */
+  public function getSubLibrary() {
+    return $this->subLibrary;
+  }
+
+  /**
+   * @param mixed $subLibrary
+   */
+  public function setSubLibrary($subLibrary) {
```

Comment:
Could the visibility of the setters here be reduced to `protected`? Once we have a holding group I do not think we want outsiders to change the properties.

`protected` or even `private` may be usable within our factory method.